packages/date-picker/test/basic.test.js

```diff
@@ -539,13 +539,6 @@ describe('clear button', () => {
     expect(datepicker.value).to.equal('');
   });
 
-  it('should validate on clear', () => {
```

Comment:
This was tested in two different files. Removed this test and moved the other one to `validation.test.js`